lib/goose/dbconf.go

```diff
@@ -121,9 +122,15 @@ func NewDBConf(dbDir, env string) (*DBConf, error) {
 		drv = imprt[i+1:]
 	}
 
-	open, _ := confGet(f, env, "open")
+	dsn, err := confGet(f, env, "dsn")
+	if _, ok := err.(*yaml.NodeNotFound); ok {
+		log.Println("WARNING: Database 'dsn' not specified. Please check ensure " +
+			"that you have a 'dsn' node set in your dbconf file. If you " +
+			"are not using a conf file please ensure that you have set " +
+			"the 'DB_DSN' environment variable.")
```

Comment:
We might not want this. An empty (not present) DSN is a valid configuration. For example sqlite will use an in-memory database. Postgresql will connect to localhost with the same username & db name as the current user (if I recall correctly).